misc/psi/tests/lookup.bats

```diff
@@ -47,7 +47,7 @@ function teardown {
   $decrypt ${data_prefix}/${prefix_bgv}.sk result.ctxt -o "result.ptxt"
   $decrypt ${data_prefix}/${prefix_bgv}.sk result.ctxt_and -o "result.ptxt_and"
   $decrypt ${data_prefix}/${prefix_bgv}.sk result.ctxt_or -o "result.ptxt_or"
-  $decrypt ${data_prefix}/${prefix_bgv}.sk result.ctxt_expand -o "result.ptxt_expand"
+  $decrypt ${data_prefix}/${prefix_bgv}.sk result.ctxt_expand -o "result.ptxt_expand"  
```

Comment:
Why is there now whitespace at the end of this line?

Reply:
Yeah, I think you can revert this change.

Reply:
I removed the whitespace so it's now an unchanged file